components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3004,27 +3033,29 @@ subroutine cloud_water_autoconversion(rho,qc_incld,nc_incld,    &
    real(rtype), intent(in) :: rho
    real(rtype), intent(in) :: qc_incld
    real(rtype), intent(in) :: nc_incld
+   real(rtype), intent(in) :: qc_relvar
 
    real(rtype), intent(out) :: qcaut
    real(rtype), intent(out) :: ncautc
    real(rtype), intent(out) :: ncautr
 
-   real(rtype) :: dum
+   real(rtype) :: dum, sbgrd_var_coef
 
 #ifdef SCREAM_CONFIG_IS_CMAKE
    if (use_cxx) then
       call cloud_water_autoconversion_f(rho,qc_incld,nc_incld,    &
-         qcaut,ncautc,ncautr)
+         qc_relvar,qcaut,ncautc,ncautr)
       return
    endif
 #endif
 
    qc_not_small: if (qc_incld.ge.1.e-8_rtype) then
 
       !Khroutdinov and Kogan (2000)
-      dum   = qc_incld
-      qcaut = 1350._rtype*bfb_pow(dum,2.47_rtype)*bfb_pow(nc_incld*1.e-6_rtype*rho,-1.79_rtype)
-      ! note: ncautr is change in Nr; ncautc is change in Ncs
+      !print*,'p3_QcAutoCon_Expon = ',p3_QcAutoCon_Expon
+      sbgrd_var_coef = subgrid_variance_scaling(qc_relvar, 2.47)
```

Comment:
@PeterCaldwell 
I was about to send the PR with the 'qc_relvar' fix, but if you already done it, the compiler complains also at "`2.47`" needed to be "2.47_rtype"